core/tests/release_sources/invalid_release_mail_message.txt

```diff
@@ -0,0 +1,13 @@
+Hi all,
+   The main changes in this release are [Add main changes].
```

Comment:
The indentation seems a bit weird. Could we add a newline above this one, and drop all the left indentation? Additionally, the stuff from lines 8-10 should all go on one line (it will wrap around in the email).

Reply:
Done.

Reply:
Not done? (I see it's done in "valid_release_mail_message", but invalid_release_mail_message should probably differ from that in a more nontrivial way (perhaps try to have the body text explain why the message is invalid. Otherwise it's not obvious that it's the indentation that is causing the invalid-ness.)

Reply:
Oh, sorry I missed it. Updated now.